mmv1/products/discoveryengine/DataStore.yaml

```diff
@@ -0,0 +1,168 @@
+  ]
+
+autogen_async: true
+async: !ruby/object:Api::OpAsync
+  operation: !ruby/object:Api::OpAsync::Operation
+    path: "name"
+    base_url: "{{op_id}}"
+    wait_ms: 1000
+    timeouts: !ruby/object:Api::Timeouts
+      insert_minutes: 60
+      update_minutes: 60
+      delete_minutes: 60
+  result: !ruby/object:Api::OpAsync::Result
+    path: "response"
+  status: !ruby/object:Api::OpAsync::Status
+    path: "done"
+    complete: true
+    allowed:
+      - true
+      - false
+  error: !ruby/object:Api::OpAsync::Error
+    path: "error"
+    message: "message"
+
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    name: "discoveryengine_datastore_structured_basic"
+    primary_resource_id: 'structured_basic'
+    primary_resource_name:
+      'fmt.Sprintf("tf_test_data_store%s", context["random_suffix"])'
+    vars:
+      data_store_id: "data-store-id"
+  - !ruby/object:Provider::Terraform::Examples
+    name: "discoveryengine_datastore_unstructured_basic"
+    primary_resource_id: 'unstructured_basic'
+    primary_resource_name:
+      'fmt.Sprintf("tf_test_data_store%s", context["random_suffix"])'
+    vars:
+      data_store_id: "data-store-id"
+  - !ruby/object:Provider::Terraform::Examples
+    name: "discoveryengine_datastore_website_basic"
+    primary_resource_id: 'website_basic'
+    primary_resource_name:
+      'fmt.Sprintf("tf_test_data_store%s", context["random_suffix"])'
+    vars:
+      data_store_id: "data-store-id"
+  - !ruby/object:Provider::Terraform::Examples
+    name: "discoveryengine_datastore_advanced_site_search_basic"
+    primary_resource_id: 'advanced_site_search_basic'
+    primary_resource_name:
+      'fmt.Sprintf("tf_test_data_store%s", context["random_suffix"])'
+    vars:
+      data_store_id: "data-store-id"
+parameters:
+  - !ruby/object:Api::Type::String
+    name: 'location'
+    required: true
+    immutable: true
+    url_param_only: true
+    description: |
+      The geographic location where the data store should reside.
+  - !ruby/object:Api::Type::String
+    name: 'dataStoreId'
+    required: true
+    immutable: true
+    url_param_only: true
+    description: |
+      The unique id of the data store.
+  - !ruby/object:Api::Type::Boolean
+    name: 'createAdvancedSiteSearch'
+    default_value: false
+    url_param_only: true
+    description: |
+      If true, an advanced data store for site search will be created. If the
+      data store is not configured as site search (GENERIC vertical and
+      PUBLIC_WEBSITE contentConfig), this flag will be ignored.
+
+properties:
+  - !ruby/object:Api::Type::String
+    name: 'name'
+    description: |
+      The unique full resource name of the data store. Values are of the format
+      `projects/{project}/locations/{location}/collections/{collection_id}/dataStores/{data_store_id}`.
+      This field must be a UTF-8 encoded string with a length limit of 1024
+      characters.
+    immutable: true
```

Comment:
If this is output-only, you shouldn't need to mark it as immutable.

Reply:
Thanks. Removed it.